core/Piwik.php

```diff
@@ -889,4 +889,14 @@ public static function translate($translationId, $args = array(), $language = nu
 
         return $translator->translate($translationId, $args, $language);
     }
+
+    public static function getPeriod($default = null)
```

Comment:
We should add useful code comments for new methods. Guess something like 
```suggestion
    /**
     * Returns the period provided in the current request.
     * If no $default is provided, this method will throw an Exception if `period` can't be found in the request
     * 
     * @param string|null $default  default value to use
     * @throws Exception
     * @return string
     * @api
     */
    public static function getPeriod($default = null)
```